### PR TITLE
sprout icon on comments authored by new users

### PR DESCRIPTION
**Pull request by s-cheng**
This PR adds a sprout icon next to comment authors who are new to the forum. Currently this includes accounts that are either < 50 karma, or < 1 week old. [Figma designs here](https://www.figma.com/file/Y9VGc5yC2RqKCYS6N4cSaN/Q1.2-2023?node-id=798-18780&t=ojhG2ar6TpUDhr6h-0).

Screen Shot 2023-03-26 at 4 44 12 PM

-----
Similar on LW, with a slightly different color:
Screen Shot 2023-03-25 at 12 39 27 AM

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204266813812807) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/comments/SingleLineComment.tsx [hunk 1]
```diff
@@ -187,7 +187,7 @@ const SingleLineComment = ({treeOptions, comment, nestingLevel, parentCommentId,
         {!hideKarma && <span className={classes.karma}>
           {commentGetKarma(comment)}
         </span>}
-        <CommentUserName comment={comment} simple={true} className={classes.username} />
+        <CommentUserName comment={comment} simple={true} hideSprout className={classes.username} />
```
Comment (s-cheng) [inline packages/lesswrong/components/comments/SingleLineComment.tsx:190]: I figured it's not worth displaying the icon when the comment is so truncated that you can't really read it anyway

File: packages/lesswrong/themes/siteThemes/eaTheme.ts [hunk 2]
```diff
@@ -65,6 +65,7 @@ export const eaForumTheme: SiteThemeSpecification = {
     individual: '#BF577D',
     icon: {
       navigationSidebarIcon: shadePalette.greyAlpha(0.5),
+      sprout: '#76a47e'
```
Comment (agnesstenlund) [inline packages/lesswrong/themes/siteThemes/eaTheme.ts:68]: thanks! Changing it to a slightly more clear green so it doesn't look grey at a glance. Still not as prominent as in Figma though 
```suggestion
      sprout: '#5EB25C'
```

**Review by Will-Howard — APPROVED** [review]
🦭

**Comment by imbenclifford** [comment]
I would be happy if we kept it as it is and didn't include a max age. Why?
- We made the call to include users with <50 karma because we decided we care more about communicating this than we care about newness in terms of time.
- Leaving the symbol on old comments is consistent with that. It means you haven't engaged with the forum much.
- It's helpful context for people reading old posts that the person who commented is/was not very involved in the forum (and perhaps not the community)

That said, I don't care about this very much and if JP and Agnes feel strongly + Sarah says it's easy I'm happy to go with the 3 month cap. This is because I think the vast majority of views are on posts that are < 3 months and an even larger majority of comments that people would want to respond to are < 3 months old, so I don't think it matters that much.

I would flag to JP and Agnes that even if we add the max age, there might still be confusion about why an author who has been on the forum for years is getting a "new user" flag (particularly from the author). If you are worried about that one solution is to slightly reword the tooltip to say "Test8 hasn't used the forum very much." or something.

@agnesstenlund and @jpaddison3 do you still feel strongly?
@s-cheng is adding the max age much extra work?

**Comment by s-cheng** [comment]
@imbenclifford It's easy to add a max age, though I also agree with your take (that it's fine as-is, but don't feel strongly about it).

**Comment by jpaddison3** [comment]
My object level take remains strong.

My meta-level take is: it's definitely is your call, I don't have strong Head of Online feelings about it at all.

**Comment by agnesstenlund** [comment]
I still feel strongly that if we call someone "new" it has to involve max time, or it will be too confusing. I do agree it could be useful to see that someone who commented 1 year ago still has very low karma, but feel weird calling these users "new" if we're mainly using karma to determine this.

> "Test8 hasn't used the forum very much."

+1, would work not having a max time if we change the tooltip to not be about new-ness. "Hasn't used much" isn't necessarily true though since they could have been reading the forum forever. But something like "Test8 don't have much karma yet", or even "Test8 is either new to the forum or don't have much karma yet" could work.

probably still prefer the simple concept of newness over karma, but if what we _actually_ care about is karma (which it seems like?), a tiny rebrand would probably do the trick?

**Comment by imbenclifford** [comment]
Ok helpful, thanks JP and Agnes. Also thanks Sarah.

Yeah, what we actually care about is karma. We essentially want to say: maybe don't trust this person as much or don't think that this person is super representative of EA.

I like: "Test8 is either new to the forum or doesn't have much karma yet".

Curious if @jpaddison3 has a strong opinion against this change in wording as the solution?

**Comment by jpaddison3** [comment]
(Agree that we should ship fast.)

**Comment by imbenclifford** [comment]
Ok let's ship it then @s-cheng

File: packages/lesswrong/components/comments/CommentsItem/CommentUserName.tsx [hunk 3]
```diff
@@ -16,34 +21,71 @@ const styles = (theme: ThemeType): JssStyles => ({
...
+    return <>
+      <UsersName
+        user={author}
+        simple={simple}
+        className={classNames(className, classes.author)}
+        tooltipPlacement="bottom-start"
+      />
+      {showSproutIcon && !hideSprout && <LWTooltip
+          placement="bottom-start"
+          title={`${author.displayName} is new on ${siteNameWithArticleSetting.get()}. Take care in replying.`}
```
Comment (agnesstenlund) [inline packages/lesswrong/components/comments/CommentsItem/CommentUserName.tsx:75]:
```suggestion
          title={`${author.displayName} is either new on ${siteNameWithArticleSetting.get()} or don't have much karma yet. Take care in replying.`} 
```

**Pull request merged**